docs/sources/operations/storage/wal.md

```diff
@@ -91,7 +91,7 @@ After hitting the endpoint for `ingester-2 ingester-3`, scale down the ingesters
 
 ### Kubernetes hacking
 
-Statefulsets are significantly more cumbersome to work with/upgrade/etc. Much of this stems from immutable fields on the specification. For example, if one wants to start using the WAL with single store Loki and wants separate volume mounts for the WAL and the boltdb-shipper, you may see immutability errors when attempting updates the Kubernetes statefulsets.
+Statefulsets are significantly more cumbersome to work with, upgrade, etc. Much of this stems from immutable fields on the specification. For example, if one wants to start using the WAL with single store Loki and wants separate volume mounts for the WAL and the boltdb-shipper, you may see immutability errors when attempting updates the Kubernetes statefulsets.
```

Comment:
nit: because it's unrelated to our PR. Google style discourages etc. https://developers.google.com/style/word-list#etc